docs/telemetry.md

```diff
@@ -1,99 +1,157 @@
 # Telemetry
 
-Graphistry services can be configured to export telemetry information using the [OpenTelemetry](https://opentelemetry.io/) standard. This includes logs, metrics, and traces.
+## Overview
 
-When enabled, telemetry sent to the local Graphistry [OTEL Collector](https://opentelemetry.io/docs/collector/) service, and from there, dispatches to any observability tool that is compatible with the OpenTelemetry standard (e.g. Prometheus, Jaeger, Grafana Cloud, etc.).
+Graphistry services export telemetry information (metrics and traces) using the [OpenTelemetry](https://opentelemetry.io/) standard.
 
-To use telemetry with Graphistry, you need to:
+Graphistry services push their telemetry data to the [opentelemetry-collector](https://opentelemetry.io/docs/collector/) service (alias `otel-collector`) and this will forward the data to any observability tool that is compatible with the OpenTelemetry standard (e.g. Prometheus, Jaeger, Grafana Cloud, etc.).
 
-* Enable telemetry collection recording in one or more Graphistry services
-* Configure which users each service will monitor (see section on feature flags)
-* Enable the telemetry collector service
-* Configure the forwarding from the telemetry collector to an observability database/UI
-* Optionally, setup a local observability database/UI
+## Usage
+
+By default, the telemetry services are disabled. To enable, set `ENABLE_OPEN_TELEMETRY=true` in `$GRAPHISTRY_HOME/data/config/telemetry.env`.
+
+The `./release` script in $GRAPHISTRY_HOME will automatically start those services when `ENABLE_OPEN_TELEMETRY=true`.  The `./release` script is anologous to `docker compose ...`, but will not trigger telemetry collection, forwarding, nor telemetry UIs unless ENABLE_OPEN_TELEMETRY is set to true and the services are properly configured. 
+
+To start the otel services, run:
+```
+cd $GRAPHISTRY_HOME
+./release up -d
+```
+
+Once the services are online, we can access these links for development and operations:
+
+* OTEL Collector metrics for Prometheus: https://$GRAPHISTRY_HOST:8889/metrics
```

Comment:
I think we want to remove the port number here with the caddy changes

Reply:
Good catch, done!